app/views/shared/steps/_broken-links-summary.html.erb

```diff
@@ -4,12 +4,12 @@
   <div class="step-broken-links__summary">
     <% if step.broken_links? %>
       <span class="app-broken-link-status app-broken-link-status--fail">
-        <%= render "svgs/cross.svg" %>
+        <%= inline_svg_tag "svgs/cross.svg" %>
```

Comment:
The file referenced here (app/views/svgs/_cross.svg) should move too. We should put that as `app/assets/images/cross.svg` - similar for the tick one below